WasatchNET/Spectrometer.cs

```diff
@@ -2083,6 +2191,10 @@ public virtual double[] getSpectrum(bool forceNew=false)
                     for (int px = 0; px < pixels; px++)
                         sum[px] -= dark_[px];
 
+                // this should be enough to update the cached value
+                if (readTemperatureAfterSpectrum)
+                    _ = detectorTemperatureDegC;
+

```

Comment:
where is the cache being set? Shouldn't it be here? Might be missing a cache in detectorTemperatureDegC's setter

Reply:
if so disregard

Reply:
It's set in the last line of the Spectrometer.detectorTemperatureDegC getter.